book-crawler/app/crawler/kyobo.py

```
@@ -5,52 +5,336 @@
+        """)
+        
+        # 리뷰 수집 (여러 페이지)
+        all_reviews = []
+        max_review_pages = int(os.getenv("MAX_REVIEW_PAGES", "5"))
+        
+        # 리뷰 탭 클릭 (필요한 경우)
+        try:
+            review_tab = driver.find_element(By.XPATH, "//a[contains(@class, 'tab_link') and contains(., '리뷰')]")
+            driver.execute_script("arguments[0].click();", review_tab)
+            time.sleep(2)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)